blueprints/helper-addon/index.js

```diff
@@ -0,0 +1,3 @@
+/*jshint node:true*/
+
+module.exports = require('ember-cli/blueprints/addon-import');
```

Comment:
solved it like this: https://github.com/Turbo87/ember.js/commit/a12a1933812b58f31a9f801e004bd6329ca7b438